Tests/SwiftLayoutTests/ReferenceTests.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  ReferenceTests.swift
+//  
+//
+//  Created by maylee on 2022/03/05.
+//
+
+import XCTest
+import SwiftLayout
+
+class ReferenceTests: XCTestCase {
+
+    override func setUpWithError() throws {
+        
+    }
+
+    override func tearDownWithError() throws {
+    }
+    
+    var view: SelfReferenceView?
+    weak var weakView: UIView?
+    
+    func test1JustForPrepare() {
+        DeinitView.deinitCount = 0
+        view = SelfReferenceView()
+        weakView = view
+        
+        view?.updateLayout()
+        view?.layoutIfNeeded()
+        view = nil
+    }
+    
+    func test2ForReferenceReleasing() {
```

Comment:
It seems to be an unavoidable situation because of the problem found in #133, 
but wouldn't it be dangerous to split one test into two like this?

Reply:
yes, if we keep naming for alphabetical ordering rule, its' not danger.

Reply:
but, also want more grace solution